sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/KalixRunner.scala

```diff
@@ -233,6 +244,10 @@ final class KalixRunner private[javasdk] (
 
     logJvmInfo()
 
+    // start containers if application (only possible when running locally)
+    // TODO: stop it using CoordinateShutdown
+    dockerComposeUtils.foreach(_.start())
```

Comment:
With this change, we will start the proxy container (and whatever other containers the user configures) when we run the `Main` function. That covers the starting it up from the IDE and from the plugin. 

This code won't be executed in a few situations though:

* In prod, because we don't have a docker-compose file at the root.
* If users explicitly set `kalix.dev-mode.docker-compose-file=none`.
* If users delete their `docker-compose.yml` file.
* When running integration tests using KalixTestkit, in which case the proxy is run with Testcontainers.
* When running `sbt run` or `mvn kalix:run` instead of the `runAll` variation. 

And here is the point where I think we can let it sink a little bit. 

Do we want to have `sbt run` and `mvn kalix:run` behave differently than running the `Main` function directly? 

The other alternative is to say: `sbt run` and `mvn kalix:run` behaves the same as running `Main`. Meaning, it also runs docker-compose. 
If we believe this is a good approach, then we can simply remove the `runAll`. 

And, in case someone wants to run docker-compose manually, then they will have to set `kalix.dev-mode.docker-compose-file=none`.

There is a catch though. `mvn kalix:run` is not yet released so it's easier for us to add `docker-compose` to it, but `sbt run` is just how sbt run any project. Existing users are already used to run `sbt run` + `docker-compose up`. 
This is not such a big deal though. Running `docker-compose` twice won't make it fail. The second run will simply attach to the existing container instances and start to print the logs.

Anyway, something we can let sink for a while.